rails/locale/lv.yml

```diff
@@ -104,6 +107,10 @@ lv:
         zero: "%{count} mēneši"
         one: "%{count} mēnesis"
         other: "%{count} mēneši"
+      x_years:
```

Comment:
x_years key seems inexistent from the Action View locale:

https://github.com/rails/rails/blob/f95c0b7e96eb36bc3efc0c5beffbb9e84ea664e4/actionview/lib/action_view/locale/en.yml)